src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -84,6 +85,13 @@ const ActionsListItem = ({
     ({ ethDomainId }) => ethDomainId === parseInt(toDomainId, 10),
   );
 
+  const getFormattedRole = () => {
+    const roleNameMessage = { id: `role.${role}` };
+    return `${setTo === 'true' ? 'assigned' : 'removed'} ${formatMessage(
```

Comment:
It comes out as a string due to [this](https://github.com/JoinColony/subgraph/pull/15/files#diff-24656a8181a67d58132644c18a1f3ccdf0085bfac7eb07d9912d8725a887925cR29). Perhaps I could convert to a boolean in the util function that gets the action values

Reply:
Alicja is partly right. 

I know it's coming from the subgraph as a string, that we can't do anything about since it's a limitation, be can convert it client side.

So you should type it as a string (since it's coming in as a string), then convert it to a true boolean in the app.

This was we won't have any surprises down the line.